### ISSUE TITLE
Questions about the batch size

### ISSUE THREAD

**Issue by thuxyz15** [karfly/learnable-triangulation-pytorch#23]
Hello,

I have tried to train the volumetric model but I found the batch size stored in the configuration file(which is 5) is too large for a Titan Xp gpu(12GB). I noticed that when I set the batch size to be 2, the memory of Titan Xp got exhausted. Would you possibly tell me what type of gpus did you use? And also could you please tell me how many gpus did you use and how much is the batch size on each of them?

Thank you very much.

**Comment by karfly** [comment]
Hi, @thuxyz15!

We trained all models on the single Nvidia P40 (24 Gb) GPU. Multiple GPUs can be used to reduce training time. 

I see several ways to fit the model into 12 Gb memory:
1. Downscale images (e.g. make them [256, 256]). There is an `image_shape` parameter in the config. 
2. Set batch size to 1, but there can be some issues with batchnorm.
3. Train model with fewer number of views (e.g. 3). There are `min_n_views` and `max_n_views` parameters in the config.
4. Use GPU with larger memory 😊

**Comment by thuxyz15** [comment]
@karfly Thanks a lot!

**Comment by Lijianfang6930** [comment]
I still want know how much time usually did u cost for Volumetric training and Algebraic training respectively on ur GPU.
